.github/workflows/release.yml

```diff
@@ -17,8 +17,9 @@ jobs:
       - name: build
         run: |
           version=`cat VERSION`
-          go build -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
-
+          go build -o pulsarctl-linux -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
```

Comment:
there are two variables on building `pulsarctl` - OS and ARCH. so we should use a consistent pattern for naming the built binary - `pulsarctl-<arch>-<os>`. 

For arch, please build both 386 and amd64.